features/steps/gradle_plugin_steps.rb

```diff
@@ -136,3 +147,11 @@ def valid_mapping_api?(request_body)
   assert_not_nil(request_body['buildUUID'])
   assert_not_nil(request_body['versionName'])
 end
+
+def valid_js_source_map_api?(request_body)
+  assert_equal($api_key, request_body['apiKey'])
+  assert_equal("android", request_body['platform'])
```

Comment:
Nit (would be Rubocop violation):
```suggestion
  assert_equal('android', request_body['platform'])
```